prime-angular-frontend/src/app/modules/site-registration/pages/site-management/site-management.component.html

```diff
@@ -81,6 +81,14 @@
     </div>
   </ng-template>
 
+  <ng-template #success_notification
+               let-props="props">
+    <div class="d-flex align-self-center mt-2">
+      <mat-icon class="mr-2 green-icon">{{ props.icon }}</mat-icon>
+      <p class="text-green" [innerHTML]="props.text"></p>
```

Comment:
Looks like your HTML formatting isn't working either attributes should auto-align vertically:

```html
<p class="text-green" 
   [innerHTML]="props.text"></p>
```